scripts/_compile/index.js

```diff
@@ -100,33 +108,38 @@ async function compileScss () {
     let css = rendered.css.toString();
 
     // CSS (full) -> CSS (minified)
-    let minified = new CleanCSS({
-        advanced: true,
-    }).minify(css);
+    let minified = cssMinifier.minify(css);
 
     return {
         css,
         minified: minified.styles,
     };
 }
 
+/**
+ * WARNING: `yarn compile` should NOT be used to
+ * generate production assets.
+ *
+ * Use `yarn gulp bundle --prod` instead.
+ */
 async function compileScripts () {
-    console.log('SRC: Compiling JavaScript');
-
-    // TODO: convert to npm script
-    await exec(`yarn rollup`, { cwd: CONFIG.root }, function(error, stderr){
+    console.log('Compiling JavaScript');
+    await exec(`yarn rollup`, { cwd: CONFIG.root }, function(error, stdout, stderr){
```

Comment:
Still uses `yarn rollup` (with modified `rollup.config.js` to do compilation). This will eventually be replaced with gulp logic, but was retained in order to minimize the affect on the developer experience.